### PR TITLE
{bio,tools}[foss/2022a] ResistanceGA v4.2-5, RCall v0.13.17 w/ R 4.2.1 Julia 1.9.2

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ItIsI-Orient** [comment]
@boegelbot please test @ generoso

**Comment by ItIsI-Orient** [comment]
@boegelbot please test @ generoso

**Comment by ItIsI-Orient** [comment]
@boegelbot please test @ generoso

**Comment by ItIsI-Orient** [comment]
@boegelbot please test @ generoso

**Comment by ItIsI-Orient** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
below is the error in jsc-zen2, maybe add a patch to disable the graphics test here https://github.com/JuliaInterop/RCall.jl/blob/01e006271142c62554ccabc1194680fc7f4156a3/test/basic.jl#L98

> basic
ERROR: LoadError: REvalError: Warning in png("/tmp/eb-pj6wd1kw/jl_xw3odOra1G") :
  unable to open connection to X11 display ''
Error in .External2(C_X11, paste0("png::", filename), g$width, g$height,  : 
  unable to start device PNG
Calls: png
in expression starting at /project/def-maintainers/boegelbot/Rocky8/zen2/software/RCall/0.13.17-foss-2022a-R-4.2.1-Julia-1.9.2/packages/RCall/test/basic.jl:100
in expression starting at /project/def-maintainers/boegelbot/Rocky8/zen2/software/RCall/0.13.17-foss-2022a-R-4.2.1-Julia-1.9.2/packages/RCall/test/runtests.jl:60
ERROR: Package RCall errored during testing

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/r/RCall/RCall-0.13.17-disable_gpu_test.patch [hunk 1]
```diff
@@ -0,0 +1,32 @@
+diff -ruN RCall_orig/test/basic.jl RCall/test/basic.jl
```
Comment (smoors) [inline easybuild/easyconfigs/r/RCall/RCall-0.13.17-disable_gpu_test.patch:1]: please add short description + author info?
(and don't forget to update the checksum :)

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**